### PR TITLE
improve dealii90 entry

### PR DESCRIPTION
**Pull request by tjhei**
- missing number
- nicer key
- nicer order

### CODE REVIEW

**Comment by tjhei** [comment]
if this is okay, I will update the homepage accordingly.

File: publications-2018.bib [hunk 1]
```diff
@@ -1,11 +1,12 @@
 % Encoding: US-ASCII
 
-@Article{2018_0,
-  author    = {G. Alzetta and D. Arndt and W. Bangerth and V. Boddu and B. Brands and D. Davydov and R. Gassmoeller and T. Heister and L. Heltai and K. Kormann and M. Kronbichler and M. Maier and J.-P. Pelteret and B. Turcksin and D. Wells},
+@Article{dealII90,
   title     = {The \texttt{deal.II} Library, Version 9.0},
+  author    = {G. Alzetta and D. Arndt and W. Bangerth and V. Boddu and B. Brands and D. Davydov and R. Gassmoeller and T. Heister and L. Heltai and K. Kormann and M. Kronbichler and M. Maier and J.-P. Pelteret and B. Turcksin and D. Wells},
```
Comment (davydden) [inline publications-2018.bib]: don't see what was actually changed here.

Comment (tjhei) [reply publications-2018.bib]: title in front of author

Comment (davydden) [reply publications-2018.bib]: oh, git makes this very confusing...

**Pull request merged**